### PR TITLE
nsqadmin: conf flags for formatting statsd counters and gauges

### PR DESCRIPTION
**Pull request by kesutton**
#661 

I want to reiterate that because there was a bug where the "use-statsd-prefixes" flag was not actually passed to nsqadmin, and thus useless, now would be the best time to eliminate that flag.  Unless it worked in the past, in which case I understand backwards compatibility.

### CODE REVIEW

File: nsqadmin/README.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ Read the [docs](http://nsq.io/components/nsqadmin.html)
  3. `$ gulp clean watch` or `$ gulp clean build`
  4. `$ go-bindata --debug --pkg=nsqadmin --prefix=static/build static/build/...`
  5. `$ cd ../nsqadmin && go build && ./nsqadmin`
- 6. make changes
+ 6. make changes (and repeat steps 4 and 5 to see changes)
```
Comment (mreiferson) [inline nsqadmin/README.md]: This isn't _always_ true (specifically, it's only true when you're making changes to the Go code of `nsqadmin`).  Another way of communicating this is to say `(repeat step 5 if you make changes to any Go code)`

Comment (kesutton) [inline nsqadmin/README.md]: Will update

**Comment by mreiferson** [comment]
@jehiah what do you think?

**Comment by mreiferson** [comment]
also, thanks for putting this together @kesutton!

File: nsqadmin/static/js/lib/handlebars_helpers.js [hunk 2]
```diff
@@ -53,16 +72,19 @@ var genColorList = function(typ, key) {
 
 var genTargets = function(typ, node, ns1, ns2, key) {
     var targets = [];
-    var prefix = statsdPrefix(node ? node : '*', metricType(key));
+    var nsqPrefix = nsqStatsdPrefix(node ? node : '*');
```
Comment (mreiferson) [inline nsqadmin/static/js/lib/handlebars_helpers.js]: style nitpick, can we drop the `nsq` on these variable/function names?

**Comment by mreiferson** [comment]
I think this LGTM other than the minor comment, thanks!

**Comment by kesutton** [comment]
I will fix the comment.  Did you guys come to a decision about the avlue of keeping legacy functionality that was already broken?

**Comment by mreiferson** [comment]
I'm OK with it.

**Comment by mreiferson** [comment]
Want to rebase/squash?

**Comment by kesutton** [comment]
squashed and rebased on latest

**Comment by mreiferson** [comment]
thanks

**Pull request merged**